perceval/utils/density_matrix.py

```diff
@@ -399,15 +414,15 @@ def measure(self, modes: Union[list[int], int], all_results: bool = True):
                 prob = item_list[2]
                 if prob != 0:
                     collapsed_dm = projector @ self.mat @ projector.T  # wave function collapse
-                    resulting_dm = DensityMatrix(collapsed_dm, basis)
+                    resulting_dm = DensityMatrix(collapsed_dm, basis, check_hermitian=False)
                     resulting_dm.normalize()
                     res[key_fs] = (prob, resulting_dm)
             return res
-        else:
+        else:  # TODO : separate method
```

Comment:
This TODO remains to be done?

Reply:
Actually I hesitate to remove this method. Otherwise I can put in in another method, but I don't really know how to name it and what would be the usage...